lib/lighthouse/auth/client_credentials/service.rb

```diff
@@ -51,6 +51,7 @@ def build_assertion
       # @return [Hash] body of request to get access token
       #
       def build_request_body(assertion, scopes, auth_params = {})
+        auth_params = auth_params.nil? ? {} : auth_params
         {
```

Comment:
@ahay-agile6 `rubocop -A` gave a very easy solution to this Linting error, `auth_params = {} if auth_params.nil?`. It seems too easy... makes me feel like it must be missing something?

Reply:
Seems to check out though. I guess we can assume we need to replace the value of `auth_params` _only if_ it's `nil`. Otherwise, we can let it run.